incubator/syncthing/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```

Comment:
go straight for the stable repo

Reply:
Add use of serviceAccount, see: https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for howto